src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_COUNT_SYMBOL = 6;
+    public static final int VALID_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new CustomException("The user is not registered in the network");
```

Comment:
The exception message 'The user is not registered in the network' could be misleading because at this point, we are checking if the user object is null, not if it's registered. A more accurate message would be 'User cannot be null'.